phpstan.neon

```diff
@@ -3,6 +3,7 @@ parameters:
     ignoreErrors:
           - '#Call to an undefined method Spryker\\Shared\\Kernel\\Transfer\\AbstractTransfer::getMessageAttributesOrFail\(\)#'
           - '#Call to method appPayment\(\) on an unknown class Generated\\GlueBackend\\Ide\\AutoCompletion.#'
+          - '#Call to method translator\(\) on an unknown class Generated\\GlueBackend\\Ide\\AutoCompletion.#'
```

Comment:
This should not be needed. Seems like the generator was not running

Reply:
Nope, still needed, same as the top one.

Reply:
I've checked this once again, same issue.